package.json

```diff
@@ -19,19 +19,19 @@
     "compile:lockfile": "node tasks/compile-lockfile.js"
   },
   "devDependencies": {
-    "eslint": "5.16.0",
+    "eslint": "6.1.0",
```

Comment:
should this be 5, or is it okay here?

Reply:
5 everywhere is safest.

Reply:
I believe that 6 here kills us because eslint is running/get set from the webpack.config.js file.
I think that it refers to react-script's eslint version during that portion.